### ISSUE TITLE
[Bug] Logo in "About" is screen readable

### ISSUE THREAD

**Issue by sblatz** [mozilla-mobile/fenix#7073]
We should remove it as it's not actually important for screen readers to read. 

Decision made in https://github.com/mozilla-mobile/fenix/issues/6993

**Comment by abodea** [comment]
Verified as fixed on the latest Nightly 12/11 using Samsung Galaxy S10+(Android 9).
Note that the logo cannot be selected.
